src/approaches/reinforcement_learning_approach.py

```diff
@@ -0,0 +1,92 @@
+"""A bilevel planning approach that learns NSRTs from an offline dataset, and
+continues learning options through reinforcement learning."""
+
+from typing import Callable, List, Sequence, Set
+
+from gym.spaces import Box
+
+from predicators.src import utils
+from predicators.src.approaches.base_approach import ApproachFailure, \
+    ApproachTimeout
+from predicators.src.approaches.nsrt_learning_approach import \
+    NSRTLearningApproach
+from predicators.src.settings import CFG
+from predicators.src.structs import NSRT, Dataset, GroundAtom, \
+    InteractionRequest, InteractionResult, LowLevelTrajectory, \
+    ParameterizedOption, Predicate, State, Task, Type
+
+
+class ReinforcementLearningApproach(NSRTLearningApproach):
+    """A bilevel planning approach that learns NSRTs from an offline dataset,
+    and continues learning options through reinforcement learning."""
+
+    def __init__(self, initial_predicates: Set[Predicate],
+                 initial_options: Set[ParameterizedOption], types: Set[Type],
```

Comment:
so if we're learning options in this approach, then what are these initial options going to be?

Reply:
These aren't used by this approach or NSRTLearningApproach, etc. The initial options seem to be `env.options` which are passed in during `create_approach()` called in `main.py`. 

This is included in the arguments to init here just because super().__init()___() is called repeatedly, and BaseApproach uses it to set `self._initial_options`. 

But maybe it's not great for an approach that is learning options (NSRTLearningApproach included) to get access to the env's options?

Reply:
note: this was fixed recently (#800)